docker/DockerBuild.sh

```diff
@@ -113,20 +113,19 @@ else
   DOCKERFILE="Dockerfile"
 fi
 
-TAG=""
+TAG="${MINIFI_VERSION}"
+
 if [ "${IMAGE_TYPE}" != "release" ]; then
-  TAG="${IMAGE_TYPE}-"
+  TAG="${TAG}-${IMAGE_TYPE}"
```

Comment:
Switching the order of the version and image type could break our jenkins build, as it expects the `minimal-<version>` tag when tagging and pushing the newly built image.

Reply:
I have reverted the change to `TAG` in https://github.com/apache/nifi-minifi-cpp/pull/1093/commits/7b768106e299b922df5192ecc02ff73287f69f46